test/powershell/Host/TabCompletion/TabCompletion.Tests.ps1

```diff
@@ -985,6 +985,17 @@ switch ($x)
             $res.CompletionMatches[0].CompletionText | Should -BeExactly $afterTab
         }
 
+        It "Tab completion UNC path with forward slashes" -Skip:(!$IsWindows) {
+            $homeDrive = $env:HOMEDRIVE.Replace(":", "$")
```

Comment:
It is not used now and can be removed, yes?

Reply:
Correct. The test case is needed to validate the PR's change, but `$homeDrive` is not needed anymore. My bad. Removing that.